extensions/default/src/ViewerLayout/index.tsx

```diff
@@ -285,6 +286,7 @@ ViewerLayout.propTypes = {
   commandsManager: PropTypes.instanceOf(CommandsManager),
   servicesManager: PropTypes.instanceOf(ServicesManager),
   // From modes
+  isReturnEnabled: PropTypes.bool,
```

Comment:
Um, shouldn't this be `showStudyList` instead?

Reply:
Yes, good catch, thank you.